usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -220,7 +231,12 @@ def handle(self, *args, **options):
                 'program_activity_code': get_or_create_program_activity(row['program_activity_code'])
             }
 
-            load_data_into_model(award_financial_data, row, value_map=value_map, save=True)
+            afd = load_data_into_model(award_financial_data, row, value_map=value_map, save=False)
+            afd_queue.append(afd)
+
+        Award.objects.bulk_create(award_queue.values())
+        FinancialAccountsByAwards.objects.bulk_create(afd_queue)
+        awards_cache.clear()  # but now that cache is operating for other records...
```

Comment:
I'm not sure what the comment here means. Thinking of our future selves, would we interpret this as a todo that needs addressed? Or something informational?